discovery-provider/alembic/trigger_sql/handle_save.sql

```diff
@@ -109,6 +111,59 @@ begin
       on conflict do nothing;
     end if;
 
+    -- notify followees of the favoriter who have reposted the same content
+    -- within the last month
+    if new.is_delete is false
+    and new.is_save_of_repost is true then
+    with
+        followee_save_repost_ids as (
+            select user_id
+            from reposts r
+            where
+                r.repost_item_id = new.save_item_id
+                and new.created_at - INTERVAL '1 month' < r.created_at
+                and new.created_at > r.created_at
+                and r.is_delete is false
+                and r.is_current is true
+                and r.repost_type::text = new.save_type::text
+                and r.user_id in
+                (
+                    select
+                        followee_user_id
+                    from follows
+                    where
+                        follower_user_id = new.user_id
+                        and is_delete is false
+                        and is_current is true
+                )
+        )
+    insert into notification
+      (blocknumber, user_ids, timestamp, type, specifier, group_id, data)
+      SELECT blocknumber_val, user_ids_val, timestamp_val, type_val, specifier_val, group_id_val, data_val
+      FROM (
+        SELECT new.blocknumber AS blocknumber_val,
+        ARRAY(
+          SELECT user_id
+          FROM
+            followee_save_repost_ids
+        ) AS user_ids_val,
+        new.created_at AS timestamp_val,
+        'save_of_repost' AS type_val,
+        new.user_id AS specifier_val,
+        'save_of_repost:' || new.save_item_id || ':type:' || new.save_type AS group_id_val,
```

Comment:
Do we want save of repost and repost of repost to be bundled together when being displayed to the user? If so they should have the same group_id else this is fine

Reply:
looking here https://www.figma.com/file/70Zl2Ggimen84bYfRHa0be/Discovery-Focused-Notifs?node-id=1%3A2523&t=aN9jIPVm9CLPmGic-0 i think they are separate notifications that wouldn't be bundled